### PR TITLE
Fix Memory Bank

### LINKED ISSUE #1007: Loss stuck
Hi, I am trying to run the tutorial posted here 
https://docs.lightly.ai/self-supervised-learning/tutorials/package/tutorial_moco_memory_bank.html
But my loss is stuck at 8.32 after 100 epochs
python 3.9
pytorch-lightning 1.8.1 
lightly 1.2.38

Any suggestions on how I should troubleshoot this?
Thanks in advance!

### PR DESCRIPTION
**Pull request by guarin**
### Changes

Add missing assignment after normalizing memory bank. This made the network collapse during training because the norm of the randomly initialized bank was too large. The bug was introduced in #993 (lightly v1.2.37).

### How was it tested?

Ran the [MoCo Tutorial](https://docs.lightly.ai/self-supervised-learning/tutorials/package/tutorial_moco_memory_bank.html) and verified that the loss decreases.

Closes #1007

### CODE REVIEW

**Review by IgorSusmelj — APPROVED** [review]
LGTM
